python/datasources/cdc_wonder.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+from ingestion.cdc_wonder_utils import (
+    ALL_CANCER_CONDITIONS,
+    BREAKDOWN_TO_STANDARD_BY_COL,
+    CANCERS_WITH_SEX_BREAKDOWN,
+    TMP_ALL,
+    get_float_cols,
+    load_cdc_df_from_data_dir,
+)
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util
+from ingestion import standardized_columns as std_col
+from ingestion.constants import (
+    ALL_VALUE,
+    CURRENT,
+    HISTORICAL,
+    NATIONAL_LEVEL,
+    UNKNOWN,
+    US_NAME,
+)
+from ingestion.dataset_utils import (
+    generate_pct_rel_inequity_col,
+    generate_pct_share_col_with_unknowns,
+    generate_pct_share_col_without_unknowns,
+    generate_time_df_with_cols_and_types,
+)
+from ingestion.het_types import CANCER_TYPE_OR_ALL, GEO_TYPE
+from ingestion.merge_utils import merge_state_ids
+
+
+class CdcWonderData(DataSource):
+    @staticmethod
+    def get_id():
+        return 'CDC_WONDER_DATA'
+
+    @staticmethod
+    def get_table_name():
+        return 'cdc_wonder_data'
+
+    def upload_to_gcs(self, gcs_bucket, **attrs):
+        raise NotImplementedError('upload_to_gcs should not be called for CdcWonderData')
+
+    def write_to_bq(self, dataset, gcs_bucket, **attrs):
+        demo_type = self.get_attr(attrs, 'demographic')
+        geo_level = self.get_attr(attrs, 'geographic')
+
+        df = self.generate_breakdown_df(demo_type, geo_level)
+
+        for table_type in (CURRENT, HISTORICAL):
+            table_name = f"{demo_type}_{geo_level}_{table_type}"

```

Comment:
trying to keep things standardized:
- can we use `time_view` as the variable instead of `table_type`
- can we use `table_id = gcs_to_bq_util.make_bq_table_id(demo_type, geo_level, time_view)` to build a standardized table id